src/components/plan/PlanForm.svelte

```diff
@@ -130,12 +141,93 @@
       effects.updatePlan(plan, { name: $planNameField.value }, user);
     }
   }
+
+  async function exportPlan() {
```

Comment:
Did something happen in the rebase/merge? Seeing some export/import plan code in this PR

Reply:
I rebased https://github.com/NASA-AMMOS/aerie-ui/pull/1338 since we're merging into that branch, did something else get pulled in?

Reply:
The basic export functionality already got merged in last week. We're just hiding the button right now to not expose it.